.deps/dev.md

```diff
@@ -60,6 +60,7 @@
 | [`@testing-library/dom@7.16.2`](https://github.com/testing-library/dom-testing-library) | MIT | clearlydefined |
 | [`@testing-library/jest-dom@5.10.1`](https://github.com/testing-library/jest-dom) | MIT | clearlydefined |
 | [`@testing-library/react@10.3.0`](https://github.com/testing-library/react-testing-library) | MIT | clearlydefined |
+| [`@testing-library/user-event@12.2.2`](https://github.com/testing-library/user-event) | MIT |  |
```

Comment:
Do you have a CQ or an appropriate issue for creating needed CQs?

Reply:
I'm changing version of `user-event` to one that is known to ClearlyDefined